### ISSUE TITLE
Feedback F13 (issue 10, CM): V.3.3.12 Duplicate Row

### ISSUE THREAD

**Issue by valexande** [OP-TED/ted-rdf-mapping#359]
V.3.3.12 Why does this row exist twice in the conceptual mapping

**Comment by csnyulas** [comment]
Indeed, there was an extra (duplicated) row, which is removed now (starting with CM version 7.0.3)

We also took the opportunity to updated the CM of the "**V.3.3.12 / The winner is an SME**" field, and the field "**VII.1.7.12 / The contractor is an SME**" in accordance with the suggestions in issue #382 (i.e. setting the type of the organization to `epo:Business` comes first)

![image](https://user-images.githubusercontent.com/2441046/235948925-f2ac36b7-fea0-4943-898b-7a0bd38b9789.png)

![image](https://user-images.githubusercontent.com/2441046/235948673-8b3a8e29-fa97-482f-a3c7-e6b656ebb88a.png)
